src/core/github.js

```diff
@@ -9,6 +9,44 @@ import { pub } from "core/pubsubhub";
 
 export const name = "core/github";
 
+var body = document.body;
+var head = document.head;
+var issueBase;
```

Comment:
you need to get `issueBase` from the `conf` object that is passed to the `.run()` function. Then, all this should be wrapped in a function.

```JS
function enableFileIssuesOnSelect(issueBase){
   // ... your code here...
}

// This is just an example:
function run(conf, doc, cb){
   
   // we need a way to turn off the feature, and it only works if there is an issue base.
  // and `issueBase` is pointing to github!  
   if(conf.useSelectToFileIssues && conf.issueBase && /https://github.com/.test(conf.issueBase) ){
      enableFileIssuesOnSelect(conf.issueBase)
   }
}
```

Reply:
Note that my proposal for `conf.useSelectToFileIssues` is a terrible name... can you think of something better/shorter?

Reply:
Ok lemme think

Reply:
Well `conf.issueBase` is `undefined`. Does user pass `issueBase` as a parameter in `respecConfig` or I'm thinking of using it from here https://github.com/w3c/respec/blob/34822ee20dcbbf186e81d9bfbccd66dc3d51913e/src/core/github.js#L98

Reply:
Can I use it this way ?
```javascript
if (conf.enableIssueFeature && newProps.issueBase &&
    '/https://github.com/'.search(newProps.issueBase) ) {
    enableFileIssuesOnSelect(newProps.issueBase)
  }
```